monitor/events.go

```diff
@@ -0,0 +1,89 @@
+package monitor
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"time"
+
+	"github.com/golang/glog"
+)
+
+type Event struct {
+	Name       string            `json:"event"`
+	Properties map[string]string `json:"properties"`
+}
+
+var EventsURL = "http://metrics.livepeer.org/events"
+
+func LogStreamCreatedEvent(manifestID string, createTime time.Time) {
+	e := &Event{
+		Name: "StreamCreated",
+		Properties: map[string]string{
+			"manifestID": manifestID,
+			"createTime": fmt.Sprintf(createTime.Format(time.UnixDate)),
+		},
+	}
+
+	sendPost(e)
+}
+
+func LogStreamStartedEvent(manifestID string, startTime time.Time) {
+	e := &Event{
+		Name: "StreamStarted",
+		Properties: map[string]string{
+			"manifestID": manifestID,
+			"startTime":  fmt.Sprintf(startTime.Format(time.UnixDate)),
+		},
+	}
+
+	sendPost(e)
+}
+
+func LogStreamEndedEvent(manifestID string, createTime time.Time, endTime time.Time) {
+	e := &Event{
+		Name: "StreamEnded",
+		Properties: map[string]string{
+			"manifestID": manifestID,
+			"createTime": fmt.Sprintf(createTime.Format(time.UnixDate)),
```

Comment:
Curious the reason for including `createTime` here when it isn't in `LogStreamStartedEvent` . By omitting `createTime` and reconstructing the flow by reviewing the events based on the manifest ID, we could avoid the bookkeeping around `VideoStartTimes` .